backend/api/routes/usersRoute.js

```diff
@@ -0,0 +1,32 @@
+// IMPORTS
+const USERS_MODEL = require("../models/UsersModel");
+
+// EXPRESS ROUTER
+const router = require("express").Router();
+
+// @route  GET users/
+// @desc   Gets all of the users in the database
+// @access Public
+router.get("/", async (req, res) => {
+  try {
+    const allUsers = await USERS_MODEL.getAll_users();
+    res.status(200).json(allUsers);
+  } catch (err) {
+    res.status(500).json({ msg: err });
+  }
+});
+
+// @route  GET users/:userid
```

Comment:
random....is there a reason you are using @before each comment?

Reply:
It's just a convention I'm trying out that some devs I'm watching do (TraversyMedia). We can always revert to something else.